### ISSUE TITLE
Can I have your pre-trained model?931626406@qq.com please

### ISSUE THREAD

**Issue by Lin-N-A-N** [ZhuYun97/ShuffleNetv2-YOLOv3#2]

**Comment by ZhuYun97** [comment]
I just trained it on dogvscat dataset which is a simple binary-label dataset. So if you want to use it to do transfer learning, the result is not good.

**Comment by ZhuYun97** [comment]
I trained the model around 80 epochs, but I got a bad result.

**Comment by zhyj3038** [comment]
I tried to use mobilenet v2 as backbone of YOLOv3, the train result is poor too
